source/getting-started/scripts.markdown

```diff
@@ -74,6 +74,25 @@ delay:
 # Valid formats include HH:MM and HH:MM:SS
 delay: {% raw %}'00:{{ states.input_slider.minute_delay.state | int }}:00'{% endraw %}
 ```
+### {% linkable_title Wait %}
+
+Wait until some things are complete. We support at the moment `wait_template` for waiting until a condition is `true`, see also on [Template-Trigger](/getting-started/automation-trigger/#template-trigger). It is possible to set a timeout after that will the script abort his execution. Timeout have same syntax as `delay`.
```

Comment:
Unlike `delay` where you need to specify the wait time, `wait` uses a template to determine the wait time. You can pause the execution of the script until the condition is satisfied. At the moment, we support `wait_template` for waiting until a template condition is `true`, see also on [Template-Trigger](/getting-started/automation-trigger/#template-trigger). It is possible to set a timeout after which the script will abort execution. Timeout have the same syntax as `delay`.

(quick question - will `timeout` abort the entire script as suggested above or move to the next instruction)